### PR TITLE
db migrations

### LINKED ISSUE #10: feat: Initiate and configure db migration

### PR DESCRIPTION
**Pull request by keemzyddev**

### CODE REVIEW

**Review by inidaname — CHANGES_REQUESTED** [review]
remove to babelrc config

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "type": "module",
   "scripts": {
     "start": "node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "nodemon index.js",
```
Comment (inidaname) [inline package.json]: Please remove the dev script

File: src/models/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+// Object.keys(db).forEach(modelName => {
+//   if (db[modelName].associate) {
+//     db[modelName].associate(db);
+//   }
+// });
+
+// db.sequelize = sequelize;
+// db.Sequelize = Sequelize;
+
+// module.exports = db;
```
Comment (inidaname) [inline src/models/index.js]: Remove the comments here

File: src/models/user.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+//     firstName: DataTypes.STRING,
+//     lastName: DataTypes.STRING,
+//     email: DataTypes.STRING
+//   }, {
+//     sequelize,
+//     modelName: 'User',
+//   });
+//   return User;
+// };
+
```
Comment (inidaname) [inline src/models/user.js]: Remove comments here

**Review by inidaname — CHANGES_REQUESTED** [review]

**Review by inidaname — CHANGES_REQUESTED** [review]
Also change the PR name to reflect the issue

**Review by hallydon — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (keemzyddev) [reply package.json]: i removed the dev script

**Comment by inidaname** [comment]
@keemzyddev also edit the PR title

**Review by mxhdiqaim — COMMENTED** [review]
Don't know much about what the code does, but all good :)
Look like the user model misses ```id```, IDK if that is a case :)

**Pull request merged**